policyengine_us/variables/gov/states/ca/tax/income/deductions/itemized/ca_itemized_deductions.py

```diff
@@ -12,3 +12,17 @@ class ca_itemized_deductions(Variable):
         "https://www.ftb.ca.gov/forms/2022/2022-540.pdf"
     )
     defined_for = StateCode.CA
+
+    def formula(tax_unit, period, parameters):
+        p = parameters(period).gov.irs.deductions
+        itm_deds = [
+            deduction
+            for deduction in p.itemized_deductions
+            if deduction not in ["salt_deduction"]
+        ]
+        itm_deds_less_salt = add(tax_unit, period, itm_deds)
+        property_taxes = add(tax_unit, period, ["real_estate_taxes"])
+        salt = p.itemized.salt_and_real_estate
+        cap = salt.cap[tax_unit("filing_status", period)]
+        capped_property_taxes = min_(property_taxes, cap)
+        return itm_deds_less_salt + capped_property_taxes
```

Comment:
I think California also caps itemized deductions based on AGI. We can leave that for another issue/PR though.

https://leginfo.legislature.ca.gov/faces/codes_displaySection.xhtml?lawCode=RTC&sectionNum=17077.
https://www.ftb.ca.gov/forms/2022/2022-540-ca-instructions.html (line 29 + itemized deduction worksheet)

Reply:
Done.